wp-content/themes/core/patterns/card-post-search-result.php

```diff
@@ -14,13 +14,17 @@
 <!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|40","right":"0","bottom":"var:preset|spacing|40","left":"0"}}},"className":"p-card-search-result l-clearfix","layout":{"type":"default"}} -->
 <div class="wp-block-group p-card-search-result l-clearfix" style="padding-top:var(--wp--preset--spacing--40);padding-right:0;padding-bottom:var(--wp--preset--spacing--40);padding-left:0"><!-- wp:post-featured-image {"width":"22%","align":"right","className":"p-card-search-result__image s-aspect-ratio-cover s-aspect-ratio-4-3"} /-->
 
-<!-- wp:tribe/post-type-name {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|10"}}}} /-->
+<!-- wp:paragraph {"metadata":{"bindings":{"content":{"source":"tribe/post-type-name"}}},"className":"t-category","style":{"spacing":{"margin":{"bottom":"var:preset|spacing|10"}}}} -->
+<p class="t-category" style="margin-bottom:var(--wp--preset--spacing--10)">Post Type Name Placeholder</p>
```

Comment:
I wonder if we should use realistic placeholder values and can we style the placeholder a little differently by default? Maybe a value like `Post` instead of `Post Type Name Placeholder` here, below, and in the example markup in the classes as well?

Reply:
Do you have any ideas on styling the placeholders differently? Using bindings doesn't really give us anything to target to be able to style this differently. Only thing I was noticing was `contenteditable="false"` but not sure how reliable that is - the markup seemed different depending where the binding appeared in the template.

Reply:
Yeah, I wondered about that. Let's leave it as-is for now. Hopefully Core will provide us some additional classes/hooks that could be useful in the future.